cmd/tailscale/cli/cli.go

```diff
@@ -183,8 +183,12 @@ change in the future.
 		}
 	}
 	if envknob.UseWIPCode() {
-		rootCmd.Subcommands = append(rootCmd.Subcommands, idTokenCmd)
-		rootCmd.Subcommands = append(rootCmd.Subcommands, serveCmd)
+		rootCmd.Subcommands = append(rootCmd.Subcommands,
+			idTokenCmd,
+			serveCmd,
+			loginCmd,
```

Comment:
Let's make these like "debug": present by default if you know they're there.

And then if the plan is for them to be alpha, then document them in their ShortHelp + LongHelp loudly that they're alpha. That's what @shayne is doing in his ingress CLI PR.

Reply:
Done